File: Assignment5/program1.c
////////////////////////////////////////////////////////////////
//
//  File Name   : program1.c
//  Description : Accepts Integer and returns the pattern
//  Author      : Akhilesh.P.Sonavane.
//  Date        : 17/05/2025
//
////////////////////////////////////////////////////////////////

#include<stdio.h>

////////////////////////////////////////////////////////////////
// 
//  Function Name : Pattern
//  Description   : Take a Number and Returns a Pattern 
//  Input         : Integer
//  Output        : Integer
//
////////////////////////////////////////////////////////////////

void Pattern(int iNo)
{
    int iCnt = 0;

    for (iCnt = 1; iCnt <= iNo; iCnt++)
    {
        printf("$   *\t",iCnt);
    }
    
}

int main()
{
    int iValue = 0;

    printf("Enter Number :\n");
    scanf("%d",&iValue);

    Pattern(iValue);

    return 0;
}

File: Assignment11/program1.java
////////////////////////////////////////////////////////////////
//
//  File Name   : program1.java
//  Description : Accept a Integer and Print Unique Pattern
//  Author      : Akhilesh.P.Sonavane.
//  Date        : 22/05/2025
//
////////////////////////////////////////////////////////////////

import java.util.*;

////////////////////////////////////////////////////////////////
// 
//  Class Name    : Pattern
//  Function Name : Display
//  Description   : Inputs a Integer and returns Pattern 
//  Input         : Integer
//  Output        : Character
//
////////////////////////////////////////////////////////////////

class Pattern 
{
    public void Display(int iNo)
    {
        int iCnt = 0;
        char ch = 'A';
        for(iCnt = 1; iCnt <= iNo; iCnt++, ch++)
        {
            System.out.print(ch+"\t");
        }
    }
}
public class program1
{
    public static void main(String[] args) 
    {
        int iValue = 0;
        
        Scanner sobj = new Scanner(System.in);
        
        System.out.println("Enter Number :");
        iValue = sobj.nextInt();

        Pattern pobj = new Pattern();

        pobj.Display(iValue);

        sobj.close();
    }
}
